File: static_cdn/media_root/submissions/Thebrogrramer/REMP.cpp
#include<bits/stdc++.h>   
using namespace std;
int main()
{
int n,i,j=0;
cin>>n;
long long int a[n],sum=0;
long long int b[n];
for(i=0;i<n;++i){
	
	cin>>a[i];
	sum+=a[i];
	if(a[i]%2==0)
	{
	b[j]=a[i];
	++j;
	}
}
sort(b,b+j);
int m=n/2;
if(j>m)
while(m--)
	{
		sum+=b[j-1];
	}
else
for(i=0;i<j;++i)
sum+=b[i];
cout<<sum;
}

File: static_cdn/media_root/submissions/rabada/REMP.py
l = int(input())
arr = list(map(int,input().split()))
arr1 = []
arr2 = []
for i in range(0,l):
    if(arr[i]%2==0):
        arr1.append(arr[i])
    else:
        arr2.append(arr[i])
arr1.sort()
j=0
for i in range(1,l+1):
    if(i%2 == 0 and j<len(arr1)):
        arr1[j] = arr1[j] * 2
        j = j+1
    
print(sum(arr1)+sum(arr2))